components/CustomParticles.js

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import Particles from "react-tsparticles";
+import { loadFull } from "tsparticles";
+import styles from "./CustomParticles.module.scss";
+
+function CustomParticles(props) {
+  const particlesInit = async main => {
+    console.log(main);
+    await loadFull(main);
+  };
+  return (
+    <>
+      <Particles
+        id="tsparticles"
+        className={styles.particles}
+        init={particlesInit}
+        options={{
```

Comment:
Eu aș scoate opțiunile într-o variabilă, ca să fie puțină separare între HTML/JSX și JS.

De asemenea, aș pune obiectul în [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo)

```jsx
const options = useMemo(() => ({ 
    fullScreen: ... 
}, [props.color /* Adaugi și alte dependințe aici, dacă e nevoie */]);

return <Particles ... options={options} />
````